operational_analysis/methods/plant_analysis.py

```diff
@@ -844,14 +835,14 @@ def run_AEP_monte_carlo(self):
             gross_por = fitted_model.predict(np.array(pd.concat(reg_inputs_por, axis = 1)))
 
             if self.time_resolution == 'M': # Undo normalization to 30-day months
-                gross_lt = gross_lt*self.num_days_lt/30 
+                gross_lt = gross_lt*np.tile(self.num_days_lt,self._run.num_years_windiness)/30 
                 gross_por = gross_por*self.num_days_lt/30
```

Comment:
As we were discussing in PR #105 we should update how gross_por is calculated, so it uses the new LT correction method too, and uses inputs from _reanalysis_por instead of _reanalysis_por_avg.

Reply:
@ejsimley I agree it makes sense to fix this issue here. I've taken a first look and it's a bit more complicated than what I was expecting, because we need to be careful about seasonality impacts: if we calculate gross_por simply using _reanalysis_por, we won't necessarily get a gross energy estimate that is equally distributed throughout the year (e.g. if the POR goes from December 2016 to February 2020, winter would be weighted more in gross_por, and this should be taken into account when then calculating lt_por_ratio). Let me know if you can think of an easy way to solve this.
 
In the meantime, I've noticed an issue in how lt_por_ratio was being calculated, and I've fixed it.

Reply:
@ejsimley, I've been thinking a bit more about this, and still couldn't find an easy fix. I guess one option could be trimming the POR data and calculating gross_por with the new LT approach using n*12 months. Using the old LT correction to calculate gross_por doesn't make much sense to me (and would lead to misleading results when using nonlinear ML models). The other option would be to just remove from the class the couple of lines to calculate gross_por (it's just used to calculate lt_por_ratio which is given as output of the class, not sure how important that is?), but that's not good either...

Reply:
@ejsimley I think I was able to fix this! gross_por is now calculated using the new LT approach (i.e., the regression model is applied to the raw por data, without calculating any intermediate averages), and I've deleted _reanalysis_por_avg as it is not needed anymore with the new approach. To fix the seasonality issue described above, self.groupby_time_res is applied to gross_por to calculate an average gross energy value for each month (so 12 values total), and this "average" year is then used to calculate lt_por_ratio.

Reply:
I agree this incorporates the new LT approach while avoiding seasonality issues. Looks good to me!